ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/utils.py

```diff
@@ -312,6 +299,49 @@ def _filtered_wan_policies(self) -> list:
 
         return filtered_policies
 
+    @cached_property
+    def _default_wan_policy_name(self) -> str:
+        """
+        TODO make this configurable
+        """
+        return "DEFAULT-POLICY"
+
+    @cached_property
+    def _default_policy_path_groups(self) -> list:
+        """
+        Return the list of path-groups to consider when generating a default policy with AVD
+        """
+        path_groups = [
+            path_group["name"] for path_group in self.shared_utils.wan_path_groups if not get(path_group, "excluded_from_default_policy", default=False)
+        ]
+        local_path_group_names = [path_group["name"] for path_group in self.shared_utils.wan_local_path_groups]
+        if not path_groups:
+            # all path-groups are excluded from default-policy
+            raise AristaAvdError(
+                "Empty list of path-groups to use for the AVD autogenerated default policy, either `wan_path_groups` is empty "
+                "or all path-groups have `excluded_from_default_policy` set tot `true`. Make sure that at least one path-group can be "
+                "configured for the default-policy"
+            )
+        elif not [path_group for path_group in path_groups if path_group in local_path_group_names]:
```

Comment:
local_path_group_names.intersection(path_groups) can help in readability.

Reply:
interesction is for sets

```
>>> l1 = []
>>> l1.intersection([])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'list' object has no attribute 'intersection'
```

we may go in set direction